react-app/src/ExerciseOverview/ExerciseCard.js

```diff
@@ -0,0 +1,48 @@
+import { StyleSheet, Text, View } from 'react-native'
+import React from 'react'
+import { SvgXml } from 'react-native-svg'
+
+export default function ExerciseCard({ id, title, description, image }) {
+  return (
+    <View style={styles.container}>
+      <View>
+        <Text style={styles.id}>{id}</Text>
+      </View>
+      <View style={styles.textContainer}>
+        <Text style={styles.title}>{title}</Text>
+
+        <Text style={styles.description}>{description}</Text>
+      </View>
+      <View style={styles.image}>
+        {!image == `` && <SvgXml width='100%' height='100%' xml={image} />}
```

Comment:
What does the logic here means?

Reply:
Do you mean image != ''?

Reply:
It means while there is not a empty string, it will generate a svg.